lib/va_common/version.rb

```diff
@@ -1,3 +1,3 @@
 module VaCommon
-  VERSION = "0.4.0"
+  VERSION = "0.5.0"
```

Comment:
If you didn't actually do a deploy to rubygems.org yet, I believe you can just leave this as 0.4.0 still until the next deploy?